content/pt/docs/concepts/storage/persistent-volumes.md

```diff
@@ -7,9 +7,15 @@ reviewers:
 - xing-yang
 título: Persistentes Volumes
 funcionalidade:
+<<<<<<< HEAD
```

Comment:
Your commit have merge problems, please, can you make a review all in this article?
Thanks